browser/playlist/playlist_data_source.h

```diff
@@ -38,18 +32,43 @@ class PlaylistDataSource : public FaviconSource {
   void StartDataRequest(const GURL& url,
                         const content::WebContents::Getter& wc_getter,
                         GotDataCallback got_data_callback) override;
+  void StartRangeDataRequest(const GURL& url,
+                             const content::WebContents::Getter& wc_getter,
```

Comment:
The docs for `content::WebContents::Getter` say that most usages can be replaced with a `WeakPtr` - is there any reason we use the `Getter` here (and other places in this file) instead?

Reply:
I guess content::WebContents::Getter is just a legacy that should be removed.  The `weakptr` part for the doc was updated later, leaving technical debt.

https://chromium-review.googlesource.com/c/chromium/src/+/5190999